.changeset/tricky-dodos-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@svenvw/fdm-app": minor
+---
+
+Make nutrient balance, nutrient advice, and norms pages show spinners when switching to different farms or fields.
```

Comment:
Please improve this changeset. Skeletons and spinners were already in place. However, they were not shown when navigating to a date-heavy page due to incorrect implementation. This PR fixes that and ensures smooth navigation to data-heavy pages.